### DISCUSSION TITLE
Question on Multiple account ploting and farming

### DISCUSSION THREAD

**Discussion by SssssHow** [Chia-Network/chia-blockchain#5485] [Support]
Hello, I was ploting and farming on my first 20TB plots with my first wallet account. And after that, I opened a new wallet account and began to plot and farm the second 20TB plots.(Within the same computer and same Chia Blockchain programme, I can see both wallet account on login and I choose the new wallet account to plot and farm.) Now I can see I'm farming on 40TB plots on my Chia Blockchain programme. But I'm not sure if I'm farming on both 40TB plots or acctually I'm only farming on the new 20TB plots which is linked to my new wallet account? Thank you.

**Comment by SssssHow** [comment]
upupup

**Reply by SssssHow** [reply]
upup

**Reply by SssssHow** [reply]
upupup

**Reply by SssssHow** [reply]
upupup

**Reply by SssssHow** [reply]
upupup

**Comment by tiny0little** [comment]
could you post ```chia farm summary``` output?

**Reply by SssssHow** [reply]
Farming status: Farming
Total chia farmed: 0.0
User transaction fees: 0.0
Block rewards: 0.0
Last height farmed: 0
Plot count: 543
Total size of plots: 53.750 TiB
Estimated network space: 12042.286 PiB
Expected time to win: 1 month and 2 weeks
Note: log into your key using 'chia wallet show' to see rewards for each key

**Reply by SssssHow** [reply]
Thanks for the reply. the output this like above. and the 543 Plot include the Plots from both my wallet account.

**Reply by tiny0little** [reply]
I think you should be ok

each plot has one of your ```farmer public key```, ```pool public key``` and ```reward receiving address```, you can see them via ```chia keys show```

**Comment by BrandtH22** [comment]
@SssssHow , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
